spec/features/borrower_manages_loan_request.rb

```diff
@@ -1,12 +1,18 @@
 require "rails_helper"
 
-RSpec.feature "borrower creates loan request" do
+RSpec.feature "borrower manages loan request" do
   let(:borrower) { User.create(name: "Richard",
                                email: "richard@example.com",
                                password: "password",
                                role: "borrower")
                    }
 
+  let(:lender) { User.create(name: "Rachel",
+                               email: "rachel@example.com",
+                               password: "password",
+                               role: "lender")
+                   }
```

Comment:
`end` at 14, 19 is not aligned with `let(:lender) { User.create(name: "Rachel",` at 10, 2